types/framework/entities/user.d.ts

```diff
@@ -216,6 +216,43 @@ declare class User extends Entity<User.Data> {
   protected _onDelete(options: Entity.DeleteOptions, userId: string): void;
 }
 
+/**
+ * A subclass of Set which manages the Token ids which the User has targeted
+ * @internal
+ */
+declare class UserTargets extends Set<Token> {
```

Comment:
This should go in its own file. But apart from it, it looks good.

Reply:
It's only ever used as a member of User, so I figured it was in the same category of User.Data. It's even marked `@private` in foundry.js.

Reply:
Hmm usually classes go into their own file. Only types are allowed in the same file. @ghost91- seems to agree. @am-a what do you think?

Reply:
Personally, I don't care. It's clearly marked only for internal use, so I'd be fined with leaving it where it is. But I can understand the other position, too (i.e. all classes should be in their own file).